crates/noirc_driver/src/lib.rs

```diff
@@ -178,6 +181,13 @@ impl Driver {
         CompiledProgram { circuit, abi: Some(abi) }
     }
 
+    #[cfg(not(feature = "std"))]
+    pub fn add_std_lib(&mut self) {
+        // TODO: Currently, we do not load the standard library when the program
```

Comment:
This needs an issue opened for it. It just means that currently in the browser, its not possible to use the standard library

Reply:
Issue opened for this: https://github.com/noir-lang/noir/issues/323